UI/Pages/Cookies.cshtml

```diff
@@ -66,7 +67,7 @@
         </ul>
 
         <form method="post">
-            <input asp-for="ReturnUrl" value="@Model.ReturnUrl" hidden=true />
+            <input asp-for="ReturnUrl" value="@Model.ReturnUrl" hidden=true/>
```

Comment:
The updated formatting seems a little strange in this file. What formatter are you using?

Reply:
It's the Rider equivalent to Visual studio's Control + k, Control +d